dialogue-annotations-example/src/test/java/com/palantir/myservice/example/MyServiceIntegrationTest.java

```diff
@@ -148,13 +141,11 @@ public void close() {}
     }
 
     @Test
-    @Disabled
     public void testCustomResponse200() {
         testCustomResponse(200);
     }
 
     @Test
```

Comment:
Remaining 2 tests use an endpoint that has a custom param type, which is not supported right now.